Packs/MicrosoftGraphListener/ReleaseNotes/1_3_8.md

```diff
@@ -0,0 +1,4 @@
+
+##### Microsoft Graph Mail Single User
+- **Breaking changes:** Make sure to update *Microsoft Graph Mail* pack, before this pack, to avoid losing **Microsoft Graph Single User** integration's configured instance.
```

Comment:
```suggestion
- **Breaking changes:** Make sure to update the **Microsoft Graph Mail** pack, before this pack, to avoid losing the **Microsoft Graph Single User** integration's configured instance.
```